### DISCUSSION TITLE
📢 Suspending monthly time-based releases for a while

### DISCUSSION THREAD

**Discussion by adangel** [pmd/pmd#4600] [Announcements]
Hi all,

for a very long time, we provided a new release of PMD every month (almost). But right now, not much has happened since the last release of [7.0.0-rc3](https://github.com/pmd/pmd/releases/tag/pmd_releases%2F7.0.0-rc3) - see the [changes on master since then](https://github.com/pmd/pmd/compare/pmd_releases/7.0.0-rc3...master).

Therefore I decided to skip the release this month for sure (it would be due on June 24th or June 27th) and most likely the next month in July as well. In August, we'll need to see, how far we are in terms of progress for the final 7.0.0 release (see #3898). I also want to avoid to release endless release candidates - so I'm thinking of skipping releases maybe until 7.0.0 is ready.

In case you want to try out the latest version/bugfixes, there is always a 7.0.0-SNAPSHOT build available (in [OSS Repository Hosting](https://oss.sonatype.org/content/repositories/snapshots/)) or the binary distribution on [sourceforge files](https://sourceforge.net/projects/pmd/files/pmd/7.0.0-SNAPSHOT/).

**Comment by jsotuyod** [comment]
I support this. Let's just focus on wrapping up PMD 7 and take it from there.
